prov/efa/src/rxr/rxr_init.c

```diff
@@ -125,6 +129,9 @@ static void rxr_init_env(void)
 			    &rxr_env.efa_min_read_write_size);
 	fi_param_get_size_t(&rxr_prov, "inter_read_segment_size",
 			    &rxr_env.efa_read_segment_size);
+	fi_param_get_bool(&rxr_prov, "fork_safe", &fork_safe);
+	if (fork_safe)
+		efa_fork_status = EFA_FORK_SUPPORT_ON;
```

Comment:
If RDMAV_FORK_SAFE/IBV_FORK_SAFE is set, we also want to set fork support to be on

Reply:
Right, those do get checked in `efa_check_fork_enabled` but it'd be more intuitive to check it here. Let me move that check.

Reply:
Thanks!